.github/workflows/pipeline.yaml

```diff
@@ -20,13 +21,23 @@ jobs:
       GRAPH_STORE_ENDPOINT: ${{ secrets.GRAPH_STORE_ENDPOINT }}
       GRAPH_STORE_USER: ${{ secrets.GRAPH_STORE_USER }}
       GRAPH_STORE_PASSWORD: ${{ secrets.GRAPH_STORE_PASSWORD }}
+      PUBLISH_GRAPH_STORE_ENDPOINT: ${{ secrets.GRAPH_STORE_ENDPOINT }}
```

Comment:
Isn't it going to be an issue to use the same variable names for both jobs?

Reply:
As long as we store the graph into the same graph store, as we want to do in test, as far as i know, this should not be a problem. The cube ends up in the cube namespace which is different from everything in the in cube project.
Since we will use gitlab for integ and production, I let it like this.